include/conversation.php

```diff
@@ -96,7 +96,7 @@ function item_redir_and_replace_images($body, $images, $cid) {
 function localize_item(&$item){
```

Comment:
Strange, not disappearing. :-( But it is there in my repo: https://github.com/Quix0r/friendica/blob/rewrites/coding-convention-split2-1-2/include/conversation.php#L101 cc @github